src/Assets.php

```diff
@@ -29,7 +29,8 @@ public static function init() {
 	 * Register block scripts & styles.
 	 */
 	public static function register_assets() {
-		self::register_style( 'wc-block-editor', plugins_url( 'build/editor.css', __DIR__ ), array( 'wp-edit-blocks' ) );
+		self::register_style( 'wc-packages', plugins_url( 'build/packages.css', __DIR__ ) );
```

Comment:
Can you explain what this is for? Not seen this file before.

Reply:
That came from the webpack [SplitChunksPlugin](https://webpack.js.org/plugins/split-chunks-plugin/), it has some heuristics to decide how files should be split based on the way they are used in the app. When I included the `ProductControl` in `ReviewsByProduct`, it split the styles so it could be possible to include the packages styles without including the editor ones and viceversa.

In practice, we are including always both of them, so we don't get anything from splitting those files. Fixed in 4442afdde3b371f19b378838f824acf31c556fd1.